etc/pr_check/main.go

```diff
@@ -0,0 +1,149 @@
+	flag.BoolVar(&printPRs, "print", false, "If true, print all matching PRs scanned, grouped by author, with all PRs matching the given condition starred")
+	flag.BoolVar(&status, "add-pr-status", false, "Add a commit status to all PRs scanned (a failing status for those matching --no-jira-ticket or --last-minute, and a passing status to the rest)")
+	flag.BoolVar(&comment, "comment", false, "Add a comment to any PRs identified by --no-jira-ticket or --last-minute")
+	flag.BoolVarP(&help, "help", "h", false, "Print a help message and exit.")
+}
+
+func main() {
+	// Parse and validate commandline args
+	flag.Parse()
+	if help || (!printPRs && !comment && !status) {
+		flag.PrintDefaults()
+		os.Exit(0)
+	}
+
+	var check prCheck
+	if missingJira && lastMinute {
+		fmt.Fprintf(os.Stderr, "Error: only one of --no-jira-ticket and --last-minute can be set")
+		os.Exit(1)
+	} else if missingJira {
+		check = missingJiraTicket
+	} else if lastMinute {
+		check = createdLastMinute
+	}
+
+	if (comment || status) && !(missingJira || lastMinute) {
+		fmt.Fprintf(os.Stderr, "Error: if --comment or --status is set, then one of --no-jira-ticket or --last-minute must be set")
+		os.Exit(1)
+	}
+	var action prAction
+	if comment && status {
+		fmt.Fprintf(os.Stderr, "Error: at most one of --comment or --status may be set")
+		os.Exit(1)
+	} else if comment {
+		action = commentOnPr
+	} else if status {
+		action = addStatus
+	}
+	if len(prsToScan) > 0 && (len(authors) > 0 || startDay != "" || endDay != "") {
+		fmt.Fprintf(os.Stderr, "Error: if --pr-numbers is set, then --authors, --start, and --end must not be set")
+		os.Exit(1)
+	}
+
+	// initialize start and end time (from --start and --end)
+	var end, start time.Time
+	var err error
+	if startDay != "" {
+		start, err = time.Parse(time.DateOnly, startDay)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to parse --start-day value %q (must match YYYY-MM-DD): %s", startDay, err)
+			os.Exit(1)
+		}
+	}
+	if endDay != "" {
+		end, err = time.Parse(time.DateOnly, endDay)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to parse --end-day value %q (must match YYYY-MM-DD): %s", endDay, err)
+			os.Exit(1)
+		}
+	}
+	// Clamp 'start' and 'end'. This simplifies date math by ensuring 'start' and
+	// 'end' are defined, and 'start' is before the earliest conceivable creation
+	// time of a PR, and end is after the the latest conceivable creation time of
+	// a PR for Pachyderm.
+	earliestStart := earliestPR.Add(-24 * time.Hour)
+	if start.Before(earliestStart) {
+		start = earliestStart
+	}
+	latestEnd := time.Now().Add(240 * time.Hour)
+	if end.IsZero() || end.After(latestEnd) {
+		end = latestEnd
+	}
+	if start.After(end) {
+		fmt.Fprintf(os.Stderr, "PR start day (%s) must be before or on PR end day (%s)", startDay, endDay)
+		os.Exit(1)
+	}
+	authorsMap := make(map[string]bool)
+	for _, a := range authors {
+		authorsMap[a] = true
+	}
+
+	// Initialize GitHub client
+	fmt.Println("Starting...")
```

Comment:
This should be `log.Info` after we move into src/internal/tools and can use the src/internal/log package.